test/integration/imageimporter_test.go

```diff
@@ -993,3 +1121,67 @@ const phpManifest = `{
 }`
 
 const danglingDigest = `sha256:f374c0d9b59e6fdf9f8922d59e946b05fbeabaed70b0639d7b6b524f3299e87b`
+
+const convertDigest = `sha256:2a3b2f1a74f4351d1faf7ca48ea7f0ca97fad0801053fcf16df8474af6b89229`
+const convertedManifest = `{
```

Comment:
Also make it clear that it's v2->v1 conversion (either reflect it in the name or godoc).